src/pages/profile-setting/ProfileSetting.tsx

```
@@ -41,88 +39,51 @@ export default function ProfileSetting() {
   return (
     <>
       <Navigation
         leftIcon={<Icon name='ic_back' className='text-grayscale-900' />}
         handleLeftClick={handleGoBack}
         text='프로필 설정'
+        rightIcon={
+          <Button
+            style={{ paddingLeft: '2rem', paddingRight: '2rem' }}
+            children={'편집'}
+            variant={'default'}
+            buttonStyle={'edit'}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)